File: FrontEnd Elaborated/Abhilash FrontEnd/src/transaction2.ts
export class Transaction2{

    transType:string;
    transOption:string;
    transAmount:number;
    transFrom:string;
    transTo:string;

    constructor(
        transType:string,
        transOption:string,
        transAmount:number,
        transFrom:string,
        transTo:string){
            this.transAmount=transAmount;
            this.transFrom=transFrom;
            this.transTo=transTo;
            this.transType=transType;
            this.transOption=transOption;
        }      
}

File: src/main/java/com/cg/dto/Transaction2.java
package com.cg.dto;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class Transaction2 {

	private String transType;
	private String transOption;
	private double transAmount;
	private String transFrom;
	private String transTo;
	
}
